Source/Core/Core/HW/DVD/DVDInterface.cpp

```diff
@@ -1496,7 +1496,7 @@ static void ScheduleReads(u64 offset, u32 length, const DiscIO::Partition& parti
   dvd_offset = Common::AlignDown(dvd_offset, DVD_ECC_BLOCK_SIZE);
   const u64 first_block = dvd_offset;
 
-  if (Config::Get(Config::MAIN_FAST_DISC_SPEED))
+  if (Config::Get(Config::MAIN_EMULATE_DISC_SPEED))
```

Comment:
Um, this is definitely wrong. You probably meant this?
```suggestion
  if (!Config::Get(Config::MAIN_EMULATE_DISC_SPEED))
```

Otherwise we would be enabling fast disc speed when it should be disabled and vice-versa...